packages/apps/spaces/app/organizations/[id]/components/Timeline/TimelineActions/TimelineActions.tsx

```diff
@@ -64,18 +146,27 @@ export const TimelineActions: React.FC<TimelineActionsProps> = ({ onScrollBottom
               border='1px solid var(--gray-100, #F2F4F7)'
             >
               <ComposeEmail
+                formId={formId}
                 modal={false}
-                to={[]}
-                cc={[]}
-                bcc={[]}
-                from={[]}
-                subject={''}
-                emailContent={''}
+                to={state.values.to}
+                cc={state.values.cc}
+                bcc={state.values.bcc}
+                onSubmit={handleSubmit}
+                isSending={isSending}
               />
             </Box>
           </SlideFade>
         )}
       </Box>
+      <ConfirmDeleteDialog
+        label='Remove this email?'
```

Comment:
I think you were right, discard is perhaps better. Could we change to "Discard this email?"